Error handling in python3.9
